pkg/utils/utils.go

```diff
@@ -654,3 +654,23 @@ func GetAppUidGid(pvcName string, namespace string, backup *stork_api.Applicatio
 	}
 	return uid, gid, nil
 }
+
+// CheckAndReorganizeError - Check the error message and if large resource error found, reorganize the error message and return true and error
+func CheckAndReorganizeError(err error) (bool, error) {
```

Comment:
function name sounds very heavyweight and generic, whereas it finds out only certain type of error. 
nam ka wajan thoda kam karo..